shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -2,18 +2,27 @@
 // Licensed under the MIT License.
 "use strict";
 
+// Added to help with minfication
+let prototype = "prototype";
```

Comment:
Added to support better code minification by using index lookup to access the XXXX.prototype.